tsconfig.json

```diff
@@ -2,15 +2,15 @@
   "compilerOptions": {
     "target": "ES6",
     "module": "CommonJS",
-    "lib": ["dom", "esnext"],
+    "lib": ["dom", "es6"],
     "jsx": "react",
-    "strict": true,
```

Comment:
What's the reason for removing the `strict` flag?

Reply:
Upgrading to typescript made it so that every 
```
catch(error)
```
the error is now type 'unknown', because typescript wants us to type check every error. Having strict means we have to type check every error because we can access the error.message. The JS SDK did not have this strict option so I removed it here also.